### PR TITLE
XW | Fix UA tracking on Auth pages

### PR DESCRIPTION
**Pull request by vforge**
## Description

Fix tracking on Auth pages.

## Reviewers

@bognix

### CODE REVIEW

File: front/auth/app/tracking.js [hunk 1]
```diff
@@ -6,20 +6,20 @@ import {getQueryParam} from 'common/utils/querystring';
...
-	dimensions[8] = 'authPage';
-	// IsCorporatePage
-	dimensions[15] = 'No';
-	// newAuthEntryPage
-	dimensions[10] = getQueryParam('redirect');
-
-	M.tracker.UniversalAnalytics.setDimensions(dimensions);
+	const dimensions = {
+		// Skin
+		4: 'mercury',
```
Comment (rogatty) [inline front/auth/app/tracking.js]: Broken indentation.

**Comment by bognix** [comment]
and fix this error: `Cannot initialize UA mutltiple times.`

**Comment by rogatty** [comment]
LGTM

**Comment by bognix** [comment]
lookin' good!

**Comment by rogatty** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM

File: server/app/views/_partials/tracking-ua-auth.hbs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    </script>
+{{/unless}}
+{
+// Skin
+// LoginStatus
+// Page type
+// IsCorporatePage
+// newAuthEntryPage
+10:
+}
```
Comment (rogatty) [inline server/app/views/_partials/tracking-ua-auth.hbs]: Please clean it up.

**Comment by rogatty** [comment]
LGTM

**Pull request merged**